components/dashboard/src/admin/admin.routes.ts

```diff
@@ -28,13 +28,23 @@ export function getAdminTabs(): TabEntry[] {
             title: "Blocked Repositories",
             link: "/admin/blocked-repositories",
         },
-        {
-            title: "License",
-            link: "/admin/license",
-        },
         {
             title: "Settings",
             link: "/admin/settings",
+            alternatives: getAdminSettingsMenu().flatMap((e) => e.link),
+        },
+    ];
+}
+
+export function getAdminSettingsMenu() {
+    return [
+        {
+            title: "General",
+            link: ["/admin/settings"],
+        },
+        {
+            title: "License",
+            link: ["/admin/license"],
```

Comment:
**issue**: The Admin menu becomes inactive when navigating to the license page. Could we keep it active?

Reply:
Fixed it! Was just a one-liner fix that was needed 😆
Actually, I completely missed out that updating the `getAdminTabs` function to remove the License page route might cause such an issue, as this function is being used to set the active state on the admin menu 😅.